src/pages/index.astro

```diff
@@ -47,6 +48,8 @@ export const description =
     <Spacer />
     <Schedule />
     <Spacer />
+    <Carousel />
```

Comment:
this component isn't being [hydrated](https://docs.astro.build/en/guides/framework-components/#hydrating-interactive-components) btw

make sure you use a [client directive](https://docs.astro.build/en/reference/directives-reference/)

preferably the client:idle one